docs/authentication.md

```diff
@@ -4,33 +4,46 @@ sidebar_position: 2
 
 # Authentication
 
-The MobilePay API uses API keys to authenticate requests. They are used to get unlimited access to resources in your account.
+We are providing two different options for authentification. The first one is [OAuth access token](#get-an-oauth-access-token) which is used to get authenticated and scoped access to any MobilePay merchant account. Use them when your application accesses resources on behalf of account owners. I.e. you are an integrator. Second option is [API keys](#get-an-api-key). They are used to get unlimited access to resources in your merchant account. But firstly, we need to onboard app payments product.
```

Comment:
My suggestions:
"The first one is OAuth which..."
"Use them when your application will access resources on behalf of multiple MobilePay merchant accounts, i.e. ...."
"They are used to get access to multiple resources..."

Reply:
"But firstly, we need to onboard app payments product." change to:
"But first, you need to onboard your app in our developer portal."